### PR TITLE
refactor(component): generate base node 18 version of component #43

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #37, #43

## Summary:

Rebuild component for Node 18+
Remove old Orion dependencies and replace with AuroDesignSystem

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/demo.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+<!-- The below content is automatically added from ./useCases.md -->
+The `<auro-pane>` element should be used in situations where users may:
+
+* Select a date to see available flight options
+* See multiple dates with price and availability information
+<!-- AURO-GENERATED-CONTENT:END -->
+
+## Example(s)
+
+## Basic Example
```
Comment (blackfalcon) [inline demo/demo.md:26]:
```suggestion
## Display date

Default using` date` attribute
```

File: demo/demo.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+```html
+<auro-pane date="2020-09-20"></auro-pane>
+<auro-pane date="2020-09-21" selected></auro-pane>
+<auro-pane date="2020-09-22" disabled></auro-pane>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Date and Price
```
Comment (blackfalcon) [inline demo/demo.md:49]:
```suggestion
### Display date and price

Default using both `date` and `price` attributes.
```

File: demo/demo.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+```html
+<auro-pane date="2020-09-20"></auro-pane>
+<auro-pane date="2020-09-21" selected></auro-pane>
+<auro-pane date="2020-09-22" disabled></auro-pane>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Small
```
Comment (blackfalcon) [inline demo/demo.md:72]:
```suggestion
### Use sm size

Use the `sm` attribute to main the small UI that is consistent with mobile and desktop.
```

File: demo/demo.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+<div class="exampleWrapper">
+  <custom-header display="display">Salutations World!</custom-header>
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+  ```html
+  <custom-header display="display">Salutations World!</custom-header>
+  ```
+
+</auro-accordion>
```
Comment (blackfalcon) [inline demo/demo.md:117]: This example is not working in the displayed page. 

Screen Shot 2023-08-17 at 6 05 13 PM

File: docs/partials/apiExamples.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/fixed.html) -->
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/fixed.html) -->
+<!-- AURO-GENERATED-CONTENT:END -->
+
+</auro-accordion>
```
Comment (blackfalcon) [inline docs/partials/apiExamples.md:38]: This feature should have been removed.

File: docs/partials/apiExamples.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+#### small
+
+Use the `small` attribute to render a reduced size version of `auro-pane`.
```
Comment (blackfalcon) [inline docs/partials/apiExamples.md:42]:
```suggestion
Use the `sm` attribute to render a reduced size version of `auro-pane`.
```

There is no `small` attribute?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json [hunk 7]
```diff
@@ -3,143 +3,96 @@
...
-    "  2. $ npm run serve",
-    "  3. Go to http://localhost:3001",
+    "  2. Go to http://localhost:8000",
     "================================================================================"
   ],
-  "name": "@alaskaairux/auro-pane",
-  "version": "1.2.4",
-  "description": "Auro custom auro-pane element",
-  "author": "Product design and research",
+  "name": "@aurodesignsystem/auro-pane",
```
[the remark this answers is not in the document]
Comment (jason-capsule42) [reply package.json:9]: I'll be squashing all the commits after review and putting one updated commit message. Will add it as part of that.

File: demo/demo.md (hunk 1 above)
Comment (jason-capsule42) [reply demo/demo.md:26]: The requested change would be a diversion from all recent components docs where the main demo page lists the "Basic" example as the first example.

The attributes are all explained and called out on the API page. We should not divert from this documentation pattern.

File: demo/demo.md (hunk 3 above)
Comment (jason-capsule42) [reply demo/demo.md:72]: Old documentation language is based on content when we didn't have an API page to explicitly call out each API attribute and property. The demo page can be more generic as we have done on other components.

File: demo/demo.md (hunk 2 above)
Comment (blackfalcon) [reply demo/demo.md]: Forgot to add 

```
Default using both `date` and `price` attributes.
```

File: demo/demo.md (hunk 3 above)
Comment (blackfalcon) [reply demo/demo.md:72]: Disagree. The main demo page should have descriptions as to what the demo is illustrating back to the user. This is how the demo pages always have been done. The API page is to go into additional detail and more display options. 

Please add in the descriptive text as to the demo

```
Use the `sm` attribute to main the small UI that is consistent with mobile and desktop.
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please see comments on unresolved comments.

File: docs/partials/apiExamples.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/tabindex.html) -->
+<!-- AURO-GENERATED-CONTENT:END -->
+
+</auro-accordion>
+
+</auro-accordion>
+
+### Method Examples
+
+#### `methodName`
```
Comment (jordanjones243) [inline docs/partials/apiExamples.md:142]: Does not seem like this example needs to be here.

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
I have one small change requested. I saw no other changes needed besides what Dale has already commented on.

File: docs/partials/apiExamples.md (hunk 8 above)
Comment (blackfalcon) [reply docs/partials/apiExamples.md:142]: Good catch @jordanjones243 The one method `focus()` is not represented here and I am not sure what this is?

Comment (jason-capsule42) [reply docs/partials/apiExamples.md]: Good call out @jordanjones243. I've added all the necessary code for functional focus example to work on the docsite.

File: src/style.scss [hunk 9]
```diff
@@ -220,7 +220,7 @@
   @include sm;
 }
 
-:host(.focus-visible) {
+:host(:focus) {
```
Comment (blackfalcon) [inline src/style.scss:223]: Ohhh... good catch.

File: src/style.scss [hunk 10]
```diff
@@ -3,118 +3,142 @@
...
 
+    font-family: var(--auro-font-family-default);
+    font-weight: var(--auro-text-body-default-weight);
+
+    @supports (display: grid) {
+      display: grid;
+
+      grid-template-columns: auto;
+      grid-template-rows: 1.625rem 1.25rem;
+    }
```
Comment (blackfalcon) [inline src/style.scss:50]: CSS grid is pretty well supported, not sure we need this and the fallback anymore. No hard in leaving it there, but just FYI. 

https://caniuse.com/css-grid

File: src/style.scss [hunk 11]
```diff
@@ -3,118 +3,142 @@
...
+
     @media (hover: hover) {
       &:hover {
-        background: var(--auro-color-border-active-on-light);
         border-color: var(--auro-color-border-active-on-light);
 
+        background: var(--auro-color-border-active-on-light);
```
Comment (blackfalcon) [inline src/style.scss:56]: With the latest design token release, the `-on-light` tokens were deprecated. Might be just a good opportunity here to update them now. 

https://auro.alaskaair.com/getting-started/developers/design-tokens/deprecated

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**